style.css

```diff
@@ -671,14 +799,133 @@ div {
     margin-bottom: 127px;
   }
 
-  .error {
+  /* Popup Window */
+
+  .popup-cards-container {
     display: none;
-    color: rgb(190, 54, 54);
-    width: 37%;
-    margin: 0 auto;
-    position: relative;
-    bottom: 180px;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    padding: 5px;
+    z-index: 99999;
+    overflow-y: auto;
+  }
+
+  .popup-cards {
+    border-radius: 16px;
+    border: 1px solid var(--neutrals-n-30-l, #ebecf0);
+    background: var(--white, #fff);
+    box-shadow: 0 48px 48px 0 rgba(37, 47, 137, 0.08);
+    margin: 0 24px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 12px;
+  }
+
+  .open-window {
+    display: block;
+  }
+
+  .popup-cards-title {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .popup-cards-title h2 {
+    color: var(--theme-1-neutrals-1, #091e42);
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+    margin-top: 24px;
+    margin-left: 24px;
+  }
+
+  .popup-cards-title img {
+    width: 24px;
+    height: 24px;
+    margin-top: 40px;
+    margin-right: 32px;
+  }
+
+  .popup-cards-details {
+    display: flex;
+    gap: 2%;
+    margin-left: 30px;
+    align-items: flex-end;
+  }
```

Comment:
- Great work on implementing the changes suggested by the previous reviewer to the best of your knowledge. It shows your dedication and effort toward improving the project. However, I noticed that there are still some changes missing from the suggestions provided by the previous reviewer but don't worry I will be celebrating further of how to fix this issues 😊:
 
  - [ ] To create a good user experience kindly consider giving your technologies the right `font size` as they're currently too small doesn't match the `Figma` design template. See below for more details:
  
  <table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="305" alt="Screenshot 2023-06-28 at 23 05 38" src="https://github.com/78fatemanazari/Portfolio/assets/107192076/acb50648-18d2-42d9-9e2c-b7520731d968">
</td>
<td><img width="306" alt="Screenshot 2023-06-28 at 22 59 37" src="https://github.com/78fatemanazari/Portfolio/assets/107192076/4d01a520-34de-4f10-b268-3f8a7f528cc9"></td>
</tr>
</table>